src/plugins/lua/luaplugin.h

```diff
@@ -45,23 +47,12 @@ class LuaTableWriter;
 /**
  * This plugin allows exporting maps as Lua files.
  */
-class LUASHARED_EXPORT LuaPlugin : public Tiled::WritableMapFormat
+class LuaUtilWriter
```

Comment:
When allocated on the stack (see other comment), this class isn't used anymore outside of `luaplugin.cpp`, so it can be moved from the header to the source file.

Reply:
Done